Cpp_hands_on_programming/src/ipc_receivefile.cc

```diff
@@ -0,0 +1,21 @@
+/*
+ * ipc_receivefile.c
+ *
+ *  Created on: Feb 17, 2022
+ *      Author: kt
+*/
+#include "lib/other/Prog_init.h"
+
+int main(int argc, char* argv[]) {
+	try {
+		std::shared_ptr<Ipc_info> info;
+		Prog_init start;
+		start.printInstruction();
+		info = start.checkOptions(argc, argv);
```

Comment:
Why you need ptr instead just create stack variable here? The same for ipc_sendfile.cc

Reply:
You are right, stack variable is enough here, thx!